lib/libc/db/hash/Makefile.inc

```diff
@@ -1,6 +1,8 @@
 #       from @(#)Makefile.inc	8.1 (Berkeley) 6/4/93
 # $FreeBSD$
 
+.include <src.opts.mk>
+
```

Comment:
It feels a little weird to include in here in this `Makefile.inc` rather than pulling it up to the top level.

Reply:
Yes.  I did it because that is where MACHINE_ABI is used, but I could easily move it to lib/libc/Makefile.